db/fixtures/miq_product_features.yml

```diff
@@ -1834,6 +1834,37 @@
   :feature_type: node
   :identifier: miq_ae_class_simulation
 
+# Automate Git Repositories
+- :name: GitRepository
+  :description: Git Repository
+  :feature_type: node
+  :identifier: git_repository
+  :children:
+  - :name: View
+    :description: View Git Repositories
+    :feature_type: view
+    :identifier: git_repository_view
+    :children:
+    - :name: List
+      :description: Display List of Git Repositories
+      :feature_type: view
+      :identifier: git_repository_show_list
+    - :name: Show
+      :description: Display Individual Git Repositories
+      :feature_type: view
+      :identifier: git_repository_show
+  - :name: Modify
+    :description: Modify Git Repositories
+    :feature_type: admin
+    :identifier: git_repository_admin
+    :hidden: false
+    :children:
+    - :name: Add
+      :description: Add a Git Repository
+      :feature_type: admin
+      :hidden: false
+      :identifier: git_repository_add
+
```

Comment:
Are these shown when editing a role ?  if not, only super admins can do these Rest API's.  We need to understand if there's going to be a Classic UI equivalent (so we have the common identifiers).  @mkanoor any info on classic UI enhancement ?  

If this is API only,  we'd need to present/expose in Role Edit differently.  i.e. Everything->API->collection->action/instance->action...